client/src/Faq/Faq.js

```diff
@@ -0,0 +1,89 @@
+import "./Faq.scss";
+import text from "./Faq.yaml";
+
+import {
+  StandardRouteContainer,
+  ScrollToTargetContainer,
+} from "../core/NavComponents.js";
+import {
+  LabeledTable,
+  create_text_maker_component,
+  FancyUL,
+} from "../components";
+
+const { text_maker, TM } = create_text_maker_component(text);
+
+const qa_keys = [
+  "data_source",
+  "open_source",
+  "tools",
+  "infographic_tools",
+  "creation",
+  "update_freq",
+  "raw_data",
+  "older_data",
+  "people_data_exemptions",
+  "spending_types",
+  "fte_levels",
+  "tagging_scheme",
+  "contact",
+];
+
+const FaqIndex = () => (
+  <FancyUL ul_class="faq-index">
+    {[
+      <TM
+        key="jump_to_question"
+        k="jump_to_question"
+        el="h2"
+        className="heading-unstyled"
```

Comment:
Is this better than just
`<TM style={{fontSize: x}}/>` ?
I guess h2 is fixed font size so it's more consistent

Reply:
I'm clobbering the `h2` styling with "heading-unstyled". This is more about proper semantic HTML/a11y. Distinct sections/subsection of a route should have (appropriately hierarchical) heading elements because screen readers generally have some form of heading-based navigation (jump to next heading at level/jump to next subheading, etc.).

A real hassle to have well managed headings on more dynamic pages (although I mostly make the effort), but straightforward for a page like this.

Reply:
ahh I see!